x-pack/plugins/features/server/oss_features.ts

```diff
@@ -71,6 +71,33 @@ export const buildOSSFeatures = ({ savedObjectTypes, includeTimelion }: BuildOSS
             },
           ],
         },
+        {
+          name: i18n.translate('xpack.features.ossFeatures.discoverSearchSessionsFeatureName', {
+            defaultMessage: 'Store Search Sessions',
+          }),
+          privilegeGroups: [
+            {
+              groupType: 'independent',
+              privileges: [
+                {
+                  id: 'store_search_session',
```

Comment:
I have no preference on the name whatsoever, but as a reminder: Whatever `id` we choose here **cannot be changed** for the foreseeable future.